packages/app/src/components/SimpleIcon.tsx

```
@@ -0,0 +1,8 @@
+export default function SimpleIcon({ className }: { className?: string }) {
+  return (
+    <svg width="91" height="128" viewBox="0 0 91 128" fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)